test_tablib.py

```diff
@@ -782,6 +782,22 @@ def test_tsv_export(self):
         self.assertEqual(tsv, self.founders.tsv)
 
 
+class XLSXTests(BaseTestCase):
+    def test_xlsx_import_set(self):
+        data.append(('string', 42, 21.55))
+        data.headers = ('string', 'integer', 'float')
+        _xlsx = data.xlsx
+        data.xlsx = _xlsx
+        self.assertEqual(data.dict[0]['string'], 'string')
+        self.assertEqual(data.dict[0]['integer'], 42)
+        self.assertEqual(data.dict[0]['float'], 21.55)
+
+    def test_xlsx_wrong_char(self):
+        """A bad character should not crash the export"""
+        data.append(('string', b'\x0cf'))
```

Comment:
Right now it's raising `IllegalCharacterError`:

```
Traceback (most recent call last):
  File "test_tablib.py", line 798, in test_xlsx_wrong_char
    data.xlsx
  File "/home/travis/build/jazzband/tablib/tablib/formats/_xlsx.py", line 39, in export_set
    dset_sheet(dataset, ws, freeze_panes=freeze_panes)
  File "/home/travis/build/jazzband/tablib/tablib/formats/_xlsx.py", line 142, in dset_sheet
    cell.value = unicode(col, errors='ignore')
  File "/home/travis/virtualenv/python3.7.1/lib/python3.7/site-packages/openpyxl/cell/cell.py", line 239, in value
    self._bind_value(value)
  File "/home/travis/virtualenv/python3.7.1/lib/python3.7/site-packages/openpyxl/cell/cell.py", line 215, in _bind_value
    value = self.check_string(value)
  File "/home/travis/virtualenv/python3.7.1/lib/python3.7/site-packages/openpyxl/cell/cell.py", line 182, in check_string
    raise IllegalCharacterError
openpyxl.utils.exceptions.IllegalCharacterError
```

I think that's probably the right thing to do, it will let people know what the problem is and let them react appropriately (change the character, delete it, etc.).

@cristiano2lopes, what do you think?